src/mongo_logger.ts

```diff
@@ -19,6 +19,24 @@ export const SeverityLevel = Object.freeze({
 /** @internal */
 export type SeverityLevel = (typeof SeverityLevel)[keyof typeof SeverityLevel];
 
+/** @internal */
+export const SeverityLevelMap: Map<string | number, string | number> = new Map([
+  [SeverityLevel.OFF, 0],
```

Comment:
I think it might make sense to align these to the `code` corresponding to the severity instead, described here: https://github.com/mongodb/specifications/blob/master/source/logging/logging.rst#log-severity-levels

and maybe use `Infinity` / `-Infinity` for easier comparison where we'll be determining whether or not to log something

Reply:
We may not need this data structure in the driver when we actually begin implementing log invocations depending on how severity filtering logic is handled. However, the tests do need this though to translate the severity level requested in the tests to what should be filtered, maybe we move this out of the driver? we could always copy it back if it proves useful.

Reply:
I don't see how we avoid having a numeric map considering we have to have some way of tracking which log levels to log for a given setting

Reply:
but I don't object to settling this later; if it's for tests only though, let's move it to test and still align it with the codes, if only to avoid confusion when reading the spec (since the numbers 0-8 already get assigned there)

Reply:
Yea I imagine we'll need a numeric mapping but the check for `off` could be different. Either `off=0` and the check is `currentLvl !== 0 && currentLvl >= setLvl` or you use `off=Infinity` to always make `off` larger `currentLvl >= setLvl`, or you can reverse it 😅. And I think we're not sure yet if this filtering is going to be in each severity helper or somehow determined at the setup step, we can avoid deciding now

Reply:
well, one of those reduces extra checks and reduces the confusion with log level 0 corresponding to "off" for us and to "emergency" in the spec, so, like I said, if we are going to have any sort of number map here or in the unified test, let's make that map match the spec.

Reply:
I somehow entirely missed that the actual numbers were not equal to the one's in the spec, which explains some of our confusion last week, good catch